sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/config/SecHubScanConfigProvider.java

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.owaspzapwrapper.config;
+
+import java.io.File;
+import java.io.IOException;
+
+import com.mercedesbenz.sechub.commons.TextFileReader;
+import com.mercedesbenz.sechub.commons.model.SecHubScanConfiguration;
+import com.mercedesbenz.sechub.owaspzapwrapper.cli.ZapWrapperExitCode;
+import com.mercedesbenz.sechub.owaspzapwrapper.cli.ZapWrapperRuntimeException;
+
+public class SecHubScanConfigProvider {
+    public SecHubScanConfiguration getSecHubWebConfiguration(File secHubConfigFile) {
+        TextFileReader fileReader = new TextFileReader();
+
+        if (secHubConfigFile == null) {
```

Comment:
Please make this the first statment in the method. You do not need to initialize a TextFileReader, if the sechubConfigFile is `null`.